### PR TITLE
ci(dsg-2317): employer portal setup

### PR DESCRIPTION
**Pull request by davidyoung77**
# Basics

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Styling and layout changes
- [ ] Code style changes (formatting, local variables)
- [ ] Refactoring (no functional changes)
- [x] Build or CI related changes
- [ ] Tests
- [ ] Other... Please describe:

## What changes does this PR introduce?

- Employer portal ci

## Checklist

- [ ] I have updated the documentation accordingly
- [ ] I have confirmed all acceptance criteria defined by the story have been met
- [ ] I have confirmed that Sonar Qube findings for new code are reviewed and fixed, or will be whitelisted

### CODE REVIEW

File: .ci/cloudbuild-push-request.yaml [hunk 1]
```diff
@@ -183,6 +183,23 @@ steps:
...
+  - id: "Build Employer Client"
+    name: "gcr.io/cloud-builders/docker"
+    waitFor:
+      - "Build Builder Image and Push"
+      - "Unit Tests"
+    entrypoint: "bash"
+    args:
+      - "-c"
+      - |
+        docker build -t public \
```
Comment (davidyoung77) [inline .ci/cloudbuild-push-request.yaml:195]: change

File: .ci/cloudbuild-push-request.yaml [hunk 2]
```diff
@@ -183,6 +183,23 @@ steps:
...
+                      -t ${_GAR_URL}/public:${SHORT_SHA} \
```
Comment (davidyoung77) [inline .ci/cloudbuild-push-request.yaml:196]: public

File: .ci/cloudbuild-push-request.yaml [hunk 3]
```diff
@@ -183,6 +183,23 @@ steps:
...
+                      --build-arg NODE_BUILDER=${_GAR_BUILDER_URL}/node-builder:${SHORT_SHA} \
+                      --build-arg APP-employer \
+                      --build-arg ENVIRONMENT=production \
+                      -f docker/nginx.Dockerfile .
+        docker push ${_GAR_URL}/public:${SHORT_SHA}
```
Comment (davidyoung77) [inline .ci/cloudbuild-push-request.yaml:201]: public

File: .ci/cloudbuild-push-request.yaml [hunk 4]
```diff
@@ -215,6 +232,22 @@ steps:
...
+    entrypoint: bash
+    args:
+      - "-c"
+      - |
+        gcloud deploy releases create "${REPO_NAME}-employer--${SHORT_SHA}" \
+          --project=dsgov-services \
+          --region=us-east4 \
+          --delivery-pipeline="${REPO_NAME}-employer" \
+          --gcs-source-staging-dir=gs://dsgov-application-pipeline_clouddeploy/source \
+          --skaffold-file skaffold.public.yaml \
```
Comment (davidyoung77) [inline .ci/cloudbuild-push-request.yaml:248]: public

File: .ci/helm/web/employer/env.demo.yaml [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+createIngress: false
+
+image:
+  repository: public
+  pullPolicy: Always
+  tag: latest
+
+net:
+  applicationPort: 80
+  host: public.dsgov.demo.nuvalence.io
```
Comment (davidyoung77) [inline .ci/helm/web/employer/env.demo.yaml:16]: public

File: .ci/helm/web/employer/env.dev.yaml [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+createIngress: false
+
+image:
+  repository: public
+  pullPolicy: Always
+  tag: latest
+
+net:
+  applicationPort: 80
+  host: public-dev.dsgov.demo.nuvalence.io
```
Comment (davidyoung77) [inline .ci/helm/web/employer/env.dev.yaml:16]: update

File: .ci/helm/web/values.employer.local.yaml [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+createIngress: true
+
+image:
+  repository: my-image
+  pullPolicy: Always
+  tag: 'latest'
+
+net:
+  applicationPort: 80
+  host: 'public.dsgov.test'
```
Comment (davidyoung77) [inline .ci/helm/web/values.employer.local.yaml:15]: update

**Review by adminco — APPROVED** [review]
LGTM

**Pull request merged**